Products/CMFPlone/controlpanel/browser/resourceregistry.py

```diff
@@ -93,11 +93,13 @@ def _update(self):
                 return
             if new_name in bundles:
                 IStatusMessage(self.request).addStatusMessage(
-                    _("Record name ${new_name} already taken.", new_name=new_name), "error"
+                    _("Record name ${new_name} already taken.", mapping=dict(new_name=new_name)), "error"
                 )
                 return
             record = bundles[original_name]
             del bundles[original_name]
+            # update prefix
+            record.__prefix__ = record.__prefix__.replace(original_name, new_name)
```

Comment:
@jensens I came accross this but do not exactly know if this fix is feasible for the `RecordProxy` object to simply rename the prefix ... somehow the prefix has to be aligned to the new bundle name, otherwise `KeyError`.

Reply:
huh, ok. I think it worked somehwo. Anyway, LGTM.